.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "generated"]
 	path = generated
-	url = https://github.com/madsmtm/objc2-generated.git
+	url = https://github.com/gety-ai/objc2-generated.git
```

Comment:
It should be enough to update the submodule itself below, I think you can remove this change and CI will still pass